Framework/Algorithms/test/CorrectTOFAxisTest.h

```diff
@@ -231,7 +233,7 @@ class CorrectTOFAxisTest : public CxxTest::TestSuite {
     auto alg = createCorrectTOFAxisAlgorithm();
     TS_ASSERT_THROWS_NOTHING(alg->setProperty("InputWorkspace", inputWs))
     TS_ASSERT_THROWS_NOTHING(alg->setPropertyValue("OutputWorkspace", "_unused_for_child"))
-    TS_ASSERT_THROWS_NOTHING(alg->setPropertyValue("IndexType", "Workspace Index"))
```

Comment:
Does that mean the test was not properly set and you fixed it ?

Reply:
After the change of IN4 indexing, with monitor NOT having index 0 but a high number consistent with other ILL TOF instruments, a change had to be be made to stay consistent, and the spectrum number was a better choice.

Reply:
Sorry if that is a bit pedantic, but I try to figure out what could affect other algorithms.
 
If I get it right, the purpose of these tests seems to be the control that the algorithm is correctly executed if parameters are OK, and not being executed if parameters are flawed, and you also ensure you can set correctly properties, even if the global set of properties gives are incorrect (Ei is missing, here).

Since we do not test the "Workspace Index" case anymore, we must ensure this will not be used.
=> The underlying algorithm "CorrectTofAxis" is only using by DirectILLCollectData, so that is why everything is safe?

Reply:
That's perfectly fine. The workspace index can still be tested, but now there is an offset of 1 between the two options for IN4 due to the change mentioned before. I can easily bring back setting IndexType to Workspace Index for some tests so that both are covered. I am going to push it in the next commit.

Reply:
If that is possible, yes please. I am always afraid suppressing test cases. 
For IN4, I think what you did increase clarity of code !

Reply:
That's pushed now, in those modified cases the `ReferenceSpectra` changed as mentioned.